components/e2e-tests/.tekton/serviceaccount.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pipeline
+secrets:
+  - name: redhat-appstudio-staginguser-pull-secret
```

Comment:
you should also create e2e-tests-ci.yaml under components/authentication to give permission to create this secret.

Reply:
Thanks! That's the piece of the puzzle I was missing... When I tried it on my cluster I created this secret manually with my personal creds, but I had no idea how is this handled on our staging cluster...

Reply:
@Michkov I've added QE team as pipeline-maintainers for e2e-tests namespace. PTAL

Reply:
Include the file in components/authentication/kustomization.yaml

Reply:
I feel so dumb right now... Fixed.